portals/devportal/src/main/webapp/source/src/app/components/Apis/Listing/APICards/APIThumbPlain.jsx

```diff
@@ -298,44 +298,54 @@ function APIThumbPlain(props) {
+                                                }}
+                                                open={businessOpenPopover}
+                                                anchorEl={businessAnchorEl}
+                                                anchorOrigin={{
+                                                    vertical: 'top',
+                                                    horizontal: 'right',
+                                                }}
+                                                transformOrigin={{
+                                                    vertical: 'bottom',
+                                                    horizontal: 'left',
+                                                }}
+                                                onClose={handleBusinessPopoverClose}
+                                                disableRestoreFocus
                                             >
-                                                <div style={{ display: 'flex' }}>
-                                                    <EmailIcon fontSize='small' />
-                                                    <Typography
-                                                        variant='body2'
-                                                        style={{ marginLeft: '8px' }}
-                                                    >
-                                                        {api.businessInformation.businessOwnerEmail}
-                                                    </Typography>
+                                                <div style={{
+                                                    display: 'flex',
+                                                    flexDirection: 'column',
+                                                }}
+                                                >
+                                                    <div style={{ display: 'flex' }}>
+                                                        <EmailIcon fontSize='small' />
+                                                        <Typography
+                                                            variant='body2'
+                                                            style={{ marginLeft: '8px' }}
+                                                        >
+                                                            {api.businessInformation.businessOwnerEmail}
+                                                        </Typography>
+                                                    </div>
                                                 </div>
-                                            </div>
-                                        </Popover>
+                                            </Popover>
+                                        )}
                                     </div>
                                 )}
                                 {tileDisplayInfo.showTechnicalDetails && (
```

Comment:
Does this mean the configuration to hide owner details only applies to the technical owner, but not the business owner?

Reply:
In the popovers, we display the email of the business or technical owner if the user has provided their email. A configuration check is performed above to check whether the config is enabled for business or technical owners separately.
Eg: [tileDisplayInfo.showBusinessDetails](https://github.com/wso2/apim-apps/pull/508/files#diff-62ac3ae8892cd2cf45e2c0ad989e259916044c67bc7c71530e8b98def860a460L279)

Reply:
Ack.